app/src/main/java/code/the/fuck/com/tipsview/widget/TipsView.java

```diff
@@ -92,11 +93,28 @@ public void onClick(View v) {
                     mOnClickListener.onClick(v);
                 }
                 setVisibility(GONE);
-                destroy();
             }
         });
     }
 
+    public void show(final View targetView, Bitmap tipsBitmap) {
```

Comment:
建议都用builder模式，支持链式调用~

以及targetview和bitmap分开设置更好
```suggestion
     public TipsView show(final View targetView) {
```